marlowe-integration-tests/test/Language/Marlowe/Runtime/Integration/Contract.hs

```diff
@@ -207,13 +414,15 @@ runContractTest test = runResourceT do
           { batchSize = unsafeIntToNat 10
           , contractStore
           }
-  runNoopEventT $ flip runReaderT testHandle $ race_ test runTestComponent
+  (a, _) <- runNoopEventT $ flip runReaderT testHandle $ concurrently test runTestComponent
```

Comment:
`race_` doesn't work any more, because the `contract` component spawns no long-running threads. So `concurrently` is needed instead to ensure the tests complete.